src/main.py

```diff
@@ -180,6 +186,7 @@ def open_pr(subject, body, branch, manifest_checker=None):
         repocfg = {}
 
     automerge = repocfg.get("automerge-flathubbot-prs")
+    has_broken_source = False
 
     # Enable automatic merge if extra-data is broken unless explicitly disabled
     if automerge is not False and manifest_checker:
```

Comment:
This check is redundant if you're setting another variable (`automerge` and `has_broken_source` can both be true simultaneously).

Reply:
Oh wait. Sorry. It's not redundant. It checking that automerge wasn't explicitly disabled.
Still, since the "should we force automerge" value now sits in another variable, maybe move the check somewhere else, rename the variable to something like `force_automerge`, or keep changing `automerge` value as it was before?

Reply:
Keeping them separate allows us to use different messages in each case. I've rephrased this loop and renamed the variable.